build.gradle.kts

```diff
@@ -1,3 +1,9 @@
-plugins {
-    id("org.jetbrains.intellij") version "0.3.6"
+repositories {
+    jcenter()
```

Comment:
jcenter is a superset of mavenCentral right? (I remember reading that its more performant..so lgtm)

Reply:
I believe so, yes, many claims it behaves more performing and reliable. We can always change it if we see something wrong with it.